src/info/ravencoin.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+
+import { type EdgeCurrencyInfo } from 'edge-core-js/types'
+
+import type { EngineCurrencyInfo } from '../engine/currencyEngine.js'
+import type { BcoinCurrencyInfo } from '../utils/bcoinExtender/bcoinExtender.js'
+import { imageServerUrl } from './constants.js'
+
+const bcoinInfo: BcoinCurrencyInfo = {
+  type: 'ravencoin',
+  magic: 0xd9b4bef9, // ?
+  formats: ['bip44', 'bip32'],
+  keyPrefix: {
+    privkey: 0x80, // done
+    xpubkey: 0x0488b21e,
+    xprivkey: 0x0488ade4,
+    xpubkey58: 'xpub',
+    xprivkey58: 'xprv',
+    coinType: 0 // 175?
```

Comment:
Yes, 175 for sure. See https://github.com/satoshilabs/slips/blob/master/slip-0044.md